docs/api/vega-wallet/v2-api/get-started.md

```diff
@@ -40,7 +40,9 @@ See the full set of methods that you can use in the **[Open RPC documentation](.
 The service requires the Origin (or Referrer) HTTP header to be specified in the request. This is usually handled by the browser, so you may not have to do anything, but for software that does not use one of those headers, the request will be rejected.
 :::
 
-Use `POST /api/v2/requests` to communicate with the wallet. The request body is a JSON-RPC 2.0 payload.
+Use `POST /api/v2/requests` to communicate with the wallet. The request body is a JSON-RPC 2.0 payload. 
+
+If you want to use a [long-living token](#generate-a-long-living-token), you'll need to specify it in the `Authorization` HTTP header with the scheme `VWT`. Example: `Authorization: VWT <TOKEN>`
```

Comment:
This is not just for long-living tokens. All the tokens (even for the sessions ones (the one generated through the `client.connect_wallet` calls)) will have to be transmitted through this header.